packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx

```diff
@@ -0,0 +1,99 @@
+
+## Development Build
+
+If you haven't already, prepare your project for [expo development builds](https://docs.expo.dev/develop/development-builds/installation/).
+
+## SDK Installation
+
+Add the SDK and its required dependencies to your project:
+
+```bash title=Terminal
+npx expo install @stream-io/video-react-native-sdk @stream-io/react-native-webrtc \
+   @config-plugins/react-native-webrtc react-native-device-info \
+   react-native-incall-manager react-native-svg \
+   @react-native-community/netinfo @notifee/react-native
+```
+
+So what did we install precisely?
+
+- `@stream-io/video-react-native-sdk` (SVRN) is Stream's Video SDK which contains UI components, hooks and util functions that will enable audio/video calls.
+- `@stream-io/react-native-webrtc` is a WebRTC module for React Native, SVRN depends on this dependency, it's components and utilities to render audio/video tracks and interact with the phone's media devices.
+- `react-native-device-info` is responsible for retrieving device info and sending it to the Stream Video servers.
+- `react-native-incall-manager` handles media-routes/sensors/events during an audio/video call.
+- `react-native-svg` provides SVG support to React Native, SVRN's components and it's icons are reliant on this dependency.
+- `@react-native-community/netinfo` - is used to detect the device's connectivity state, type and quality.
+- `@notifee/react-native` - is used to keep calls alive in the background on Android.
+
+### Add Stream Video SDK's setup method
+
+Once the peer dependencies are installed, a native setup method needs to be added:
+
+#### Android
+
+Add the following in your `MainApplication.java` file:
+
+<!-- vale off -->
+```java
+// highlight-next-line
+import com.streamvideo.reactnative.StreamVideoReactNative;
+
+public class MainApplication extends Application implements ReactApplication {
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    // highlight-next-line
+    StreamVideoReactNative.setup();
+    // the rest..
+  }
+}
+```
+<!-- vale on -->
+
+#### iOS
+
+Add the following in your `AppDelegate.m` or `AppDelegate.mm` file:
+
+```c
+// highlight-next-line
+#import "StreamVideoReactNative.h"
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  // highlight-next-line
+  [StreamVideoReactNative setup];
+
+  // the rest..
+}
+```
+
+## Add config plugin
+
+Add the config plugin for `react-native-webrtc` to your `app.json` file:
+
+```js
+{
+  "expo": {
+    ...
+    "plugins": [
+      [
+        "@config-plugins/react-native-webrtc",
+        {
+          "cameraPermission": "Allow $(PRODUCT_NAME) to access your camera",
+          "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone"
+        }
+      ]
+    ]
+  }
+}
+```
+
+You can now create development builds with `eas build --profile development --local`.
```

Comment:
We should add something about permissions here, either a link to keep the call alive and the native permissions page would also be fine. Currently I have no clue if we need the foreground permission is needed unless I am aware about the feature.